angular.json

```diff
@@ -252,6 +252,14 @@
               ],
               "serviceWorker": true,
               "ngswConfigPath": "ngsw-config.json"
+            },
+            "development": {
```

Comment:
I think it's quite confusing to have a `dev` and a `development` environment. Is one of them for local development? If so, could we use `local`?

Reply:
Yeah, you're right. It should be local. I will change that

Reply:
I see that nx is creating project with `development` and `develop` as a build for dev env. Maybe that would be a better option?

Reply:
Sorry, I'm not following your comment here. Are you saying that nx doing both of these things:
1. creating project with `development`
2. creating `develop` as a build for dev env

Reply:
nx creates project with `develop` and `development` as configuration option. See this:
<img width="463" alt="image" src="https://user-images.githubusercontent.com/83639710/226459182-8dc23880-256c-4871-9969-ea1b1f158a9d.png">
First one is for building for dev environment (minifying, uglifying, tree shaking etc.) like for prod, second is for starting application locally.

Reply:
Ok, gotcha. Thanks for the screenshot